components/markets/market-card/index.tsx

```diff
@@ -70,55 +155,112 @@ const MarketCard = ({
+    const sixHours = 21600000;
+    const diff = endTime - currentTime.getTime();
+    //checks if event has passed and is within 6 hours
+    return diff < sixHours && diff > 0 ? true : false;
+  };
+
+  const isVerified = () => {
+    return creation === "Advised" && status === "Proposed" ? true : false;
+  };
+
+  const infoRows = {
+    marketType: marketType,
+    endDate: endDate,
+    hasEnded: hasEnded(),
+    outcomes: outcomes.length,
+    volume: volume,
+    baseAsset: baseAsset?.toUpperCase() ?? "ZTG",
+  };
 
   return (
     <MarketCardContext.Provider value={{ baseAsset }}>
       <motion.div
-        whileHover={{ opacity: 0.7, background: "white" }}
-        whileFocus={{ opacity: 0.5, background: "white" }}
-        whileTap={{ opacity: 0.7, background: "white" }}
+        whileHover={{ opacity: 0.7 }}
+        whileFocus={{ opacity: 0.5 }}
+        whileTap={{ opacity: 0.7 }}
         data-testid={`marketCard-${marketId}`}
         style={{
           minWidth: width ? width : "100%",
           maxWidth: width ? width : "100%",
         }}
       >
         <div
-          className={`flex flex-col justify-center w-full h-full bg-anti-flash-white rounded-[10px] p-[15px] relative ${className}`}
+          className={`flex flex-col w-full h-full bg-anti-flash-white rounded-[10px] p-[15px] relative ${className}`}
         >
-          {showDetailsOverlay && (
-            <MarketCardOverlay
-              marketId={marketId}
-              outcomes={outcomes}
-              className="top-0 left-[0]"
-              onCloseIconClick={() => setShowDetailsOverlay(false)}
-            />
-          )}
-          {/* {outcomes?.length > 0 && (
-          <MoreVertical
-            className="absolute right-[10px] text-pastel-blue cursor-pointer"
-            onClick={() => setShowDetailsOverlay(true)}
-          />
-        )} */}
-          <Link href={`/markets/${marketId}`} className="flex items-center">
-            <MarketImage image={img} alt={question} />
-            <MarketCardInfo question={question} rows={infoRows} />
+          <Link
+            href={`/markets/${marketId}`}
+            className="flex flex-col flex-1 gap-2.5"
+          >
+            <div className="flex gap-2.5">
+              <MarketImage image={img} alt={question} />
+              <div className="flex flex-wrap font-medium">
+                <MarketCardTags tags={tags} />
+                {isEnding() && (
+                  <Pill value="Ends Soon" classes="bg-red-light text-red" />
+                )}
+                {isVerified() && (
+                  <Pill
+                    value="&#x2713; Verified"
+                    classes="bg-green-light text-green"
+                  />
+                )}
+              </div>
+            </div>
+            <MarketCardInfo question={question} />
+            <div className="w-full">
+              {/* don't show if market type is scalar */}
+              {!marketType?.scalar && (
```

Comment:
I think we can show something like the `ScalarPriceRange` component where we display the predicted price and the bar will show where that predicted price is relative to the bounds

Reply:
I'll follow up with Mikhail regarding designs as the current one doesn't quite match the new aesthetics. If I recall correctly I think this was going to be addressed next sprint anyways, but I've setup the existing component for now.